softlayer/stemcell/softlayer_finder_test.go

```diff
@@ -40,7 +40,7 @@ var _ = Describe("SoftLayerFinder", func() {
 			})
 		})
 
-		Context("valid stemcell ID pointing to a SL virtual disk image", func() {
+		Context("valid stemcell UUID pointing to a SL virtual disk image", func() {
 			It("returns found as false if stemcell directory does not exist", func() {
```

Comment:
Same here. Let's make the description better, use something like:

`"returns found as false if stemcell directory does not exist"`

Thanks